src/pages/STG_PanelContactRoles.page

```diff
@@ -23,6 +24,18 @@
                         <apex:outputText styleClass="slds-form-element__help" value="{!$Label.stgHelpOCRDefaultRole}" />
                     </apex:outputPanel>
                 </div>
+
+                <!-- Organizational Default OCR -->
+                <apex:outputLabel value="{!$ObjectType.npe01__Contacts_And_Orgs_Settings__c.Fields.Default_Organizational_Contact_Role__c.Label}" for="slOrgOCRDR" styleClass="slds-form-element__label" />
+                <div class="slds-form-element__control">
+                    <apex:outputField value="{!stgService.stgCon.Default_Organizational_Contact_Role__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
+                    <apex:selectList value="{!stgService.stgCon.Default_Organizational_Contact_Role__c}" multiselect="false" size="1" rendered="{!isEditMode}" id="slOrgOCRDR" html-aria-describedby="{!$Component.slOrgOCRDRHelp}" styleClass="slds-select" >
+                        <apex:selectOptions value="{!listSOContactRoles}"/>
+                    </apex:selectList>
+                    <apex:outputPanel id="slOrgOCRDRHelp" layout="block">
+                        <apex:outputText styleClass="slds-form-element__help" value="{!$Label.stgHelpOrgOCRDefaultRole}" />
+                    </apex:outputPanel>
+                </div>
```

Comment:
I don't know why, but in the Settings page, the new org setting is pushed up against the help text of the preceding setting.  This doesn't happen if you look at the bottom two settings for Honoree and Notification.  Markup looks the same, but I may have missed something.  here's a pic to show what I mean:

![image](https://user-images.githubusercontent.com/908119/37610251-5bd687fc-2b5c-11e8-9037-7d39ddb4c34c.png)

Reply:
Good catch. In order to align the spacing, I have added <br /> tags to the page.